apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -11,7 +11,7 @@ type Properties = {
 	isFluid?: boolean;
 	isPrimary?: boolean;
 	label: string;
-	onClick?: (() => void) | undefined;
+	onClick?: React.MouseEventHandler<HTMLButtonElement> | undefined;
```

Comment:
Same, do we need this change?

Reply:
But I believe you shouldn't have `handleFormSubmit` for `onClick`. The button should be of type submit and the form should have `onSubmit`